### PR TITLE
Phoenix fourth module

### PR DESCRIPTION
**Pull request by samu975**

### CODE REVIEW

**Comment by juanhurtado10** [comment]
@samu975 Please merge #11 and rebase this branch!

File: Phoenix/hello/config/test.exs [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Config
 # Run `mix help test` for more information.
 config :hello, Hello.Repo,
   username: "postgres",
-  password: "postgres",
+  password: "5550142SaMy",
```
Comment (juanhurtado10) [inline Phoenix/hello/config/test.exs]: @samu975 It is not a good practice to push secret variables to the repository. This is a security breach!

File: Phoenix/hello/lib/hello_web/controllers/comment_controller.ex [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    render(conn, "edit.html", comment: comment, changeset: changeset)
+  end
+
+  def update(conn, %{"id" => id, "comment" => comment_params}) do
+    comment = Blog.get_comment!(id)
+
+    case Blog.update_comment(comment, comment_params) do
+      {:ok, comment} ->
+        conn
+        |> put_flash(:info, "Comment updated successfully.")
```
Comment (juanhurtado10) [inline Phoenix/hello/lib/hello_web/controllers/comment_controller.ex:46]: @samu975 what does `put_flash/2` do?

File: Phoenix/hello/lib/hello_web/controllers/fallback_controller.ex [hunk 3]
```diff
@@ -0,0 +1,24 @@
+defmodule HelloWeb.FallbackController do
```
Comment (juanhurtado10) [inline Phoenix/hello/lib/hello_web/controllers/fallback_controller.ex:1]: @samu975 Please, explain how this fallback controller works.

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: Phoenix/hello/lib/hello_web/controllers/comment_controller.ex (hunk 2 above)
Comment (samu975) [reply Phoenix/hello/lib/hello_web/controllers/comment_controller.ex:46]: put flash is for writing messages in this case the message is info type and the message is "comment updated successfully"

File: Phoenix/hello/lib/hello_web/controllers/fallback_controller.ex (hunk 3 above)
Comment (samu975) [reply Phoenix/hello/lib/hello_web/controllers/fallback_controller.ex:1]: is used to handle failed events. In this case, it was used to handle an error not found and an unprocessable_entity error.

**Review by juanhurtado10 — APPROVED** [review]

**Pull request merged**